CHANGES.md

```diff
@@ -31,49 +31,51 @@ replace with next unreleased version
 - Open bugs affecting this version
 	- [Jira view of open Issues](https://codice.atlassian.net/issues/?jql=project%3DCodice%20Alliance%20AND%20issuetype%20%3D%20bug%20%20AND%20status%20%3D%20Open%20AND%20affectedVersion%20%3D%20VERSION_NUMBER%20AND%20fixVersion%20!%3D%20VERSION_NUMBER%20ORDER%20BY%20priority) affecting this version.
  -->
-## 0.2.0
-	Release Date: `unreleased`
+
+## 0.2.1
+	Release Date: pending
 
 ### NOTES
 
-- Built on [DDF v.2.10.0](https://github.com/codice/ddf/blob/master/CHANGES.md#2100)
+- Built on [DDF v.2.10.1](https://github.com/codice/ddf/blob/master/CHANGES.md#2101)
 
 ### NEW FEATURES
 
-- [CAL-111](https://codice.atlassian.net/browse/CAL-111) Add support for JPEG2000 in the nitf input transformer
-- [CAL-98](https://codice.atlassian.net/browse/CAL-98) Add udp stream generator utility to alliance sdk
-- [CAL-102](https://codice.atlassian.net/browse/CAL-102) support for consuming MGMPv2 compliant metadata
-- [CAL-119](https://codice.atlassian.net/browse/CAL-119) Improved Image Chipping
-- [CAL-93](https://codice.atlassian.net/browse/CAL-93) Implement expanded Taxonomy in Alliance
-- [CAL-117](https://codice.atlassian.net/browse/CAL-117) NSILI default port configurable in system properties
-- [CAL-87](https://codice.atlassian.net/browse/CAL-87) Update NITF Input Transformer to use expanded taxonomy
-- [CAL-94](https://codice.atlassian.net/browse/CAL-94) Update the UdpEndpoint to create child metacards with an FMV User Subject
-- [CAL-104](https://codice.atlassian.net/browse/CAL-104) add configuration for metacard security marking defaults
-- [CAL-84](https://codice.atlassian.net/browse/CAL-84) klv lat-lon pairs when just one value is error value
-	- [JIRA View of All New Features](https://codice.atlassian.net/issues/?jql=project%20%3D%20%22Codice%20Alliance%22%20and%20issuetype%20in%20%28story%2Ctask%29%20AND%20resolution%20%3D%20done%20AND%20fixVersion%20%3D%200.2.0%20ORDER%20BY%20summary%20ASC%2C%20priority)
+- new features
```

Comment:
It's more of a placeholder than a description. It should/will be replaced as soon as there is a feature issue to replace it with. 

Having that bullet blank breaks the formatting for the link below it. I'll try to populate something on that list before this PR gets merged.